src/__tests__/components/articleDescription.spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ArticleDescription from '../../components/articleDescription/ArticleDescription';
+
+describe('Article description shallow tests', () => {
```

Comment:
@Dubby20 same here "ArticleDescription component"

Reply:
The same here